### PR TITLE
[docs] edit release notes

### PR DESCRIPTION
**Pull request by TiKeil**
Add linear optimization methods

### CODE REVIEW

**Review by renefritze — CHANGES_REQUESTED** [review]
Looks like you need to rebase on master to remove the listserv refs from `CONTRIBUTING` ?

File: docs/source/release_notes/2020.2.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@ pyMOR 2020.2 (December ?, 2020)
 We are proud to announce the release of pyMOR 2020.2! Highlights of this release
 are a non-intrusive model reduction method for instationary models based on artificial
 neural networks and an extension of our available tutorials through the new reduced
-basis projection tutorial. Additionally, as part of this release we happily present
+basis projection tutorial. Furthermore, we have added tools for linear PDE-constrained
+parameter optimization including an elaborated tutorial to use them.
```
Comment (renefritze) [inline docs/source/release_notes/2020.2.rst:8]: elaborate tutorial

File: docs/source/release_notes/2020.2.rst [hunk 2]
```diff
@@ -26,11 +28,25 @@ solved. As in the stationary case, the implementation allows for several
...
+PDE-constrained parameter optimization were added. These include partial derivatives
+of the solutions and the output with respect to the parameters.
+
+
+Tutorial: Model order reduction for PDE-constrained optimization problems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The new features that were added in `[#1110] <https://github.com/pymor/pymor/pull/1110>`_
+are gathered in a new tutorial `[#1205] <https://github.com/pymor/pymor/pull/1205>`_ which
+also focuses on general questions and insights to MOR for
+linear PDE-constrained optimization method.
```
Comment (renefritze) [inline docs/source/release_notes/2020.2.rst:43]: This is maybe missing an article? "for a linear PDE..."

**Review by renefritze — CHANGES_REQUESTED** [review]

File: docs/source/release_notes/2020.2.rst (hunk 2 above)
Comment (TiKeil) [reply docs/source/release_notes/2020.2.rst]: thx

**Pull request closed without merging**